content/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -0,0 +1,295 @@
+[here](https://docs.ci.openshift.org/docs/architecture/step-registry/).
+
+### Image Publishing and Mirroring
+
+When container images are declared as release artifacts for a repository in the CI Operator configuration file under the
+`images` list (like
+[this](https://github.com/openshift/release/blob/52b67a8df670129a188f6be99c1b64ac8e85a2a5/ci-operator/config/openshift/origin/openshift-origin-master.yaml#L26-L54)),
+a synthetic `[images]` target is available for CI Operator execution that will simply build all
+release images. In order for the container images built from your repository to be published, the
+Prow job generator will configure a Prow postsubmit job that uses the CI Operator `--target=[images]` and `--promote` flags.
+Information on how to publish images to an external registry can be found in a [separate document.](TODO alvaroaleman: Link to
+https://github.com/openshift/ci-docs/pull/9 once merged).
+
+### Product builds and becoming part of an OpenShift release image
+
+Some images become part of the OpenShift release image because they are core to the platform. To be part of the release
+image, there are additional requirements (beyond those described in the previous section).  All product teams that will
+ship an image to product must ensure their image is built in OSBS at least once and published back to the nightly test
+jobs BEFORE you reference them from another component (via the image-references file), or before you set the image label
+`io.openshift.release.operator` to get automatically included.
+
+1. Ensure you have successfully published your image to the CI integration stream
+1. Follow the [ART instructions](https://mojo.redhat.com/docs/DOC-1179058) to have them build your image
+    * On the [dist-git part of the process](https://mojo.redhat.com/docs/DOC-1168290) it is critical that you ensure your component/image names match as described in the bulleted criteria
+1. Ensure a single successful build is run (sync with ART
+1. Open the PR to add your new image to another component (your operator, usually)
+1. Once the PR is merged, verify that the [nightly builds](https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/) continue to pass (usually 2-3 hours after your PR merges) and that you didn’t break the OCP CI test
+
+### Removing or renaming an image from an OpenShift release image
+
+Sometimes we rename or remove images from a given release.  To rename an image you must generally perform the following:
+
+1. Open but do not merge PRs to
+    1. openshift/release to change the name you create
+    1. to components that reference that image (in image-references)
+    1. to ocp-build-data to change the name that ART will publish as
+1. Once all PRs are reviewed and the new name is approved
+    1. merge the openshift/release PR
+    1. If this breaks, revert the openshift/release PR and delete the NEW_NAME image stream tag
```

Comment:
etc...

Reply:
etc?

Reply:
More formatting comments along the lines above